manifest.konnector

```diff
@@ -1,17 +1,17 @@
 {
   "version": "1.0.0",
-  "name": "Connector template",
+  "name": "Digitick",
   "type": "konnector",
   "language": "node",
   "icon": "icon.png",
```

Comment:
Not sure if the white text icon will work in cozy-collect.

Reply:
@CPatchane @gregorylegarec 
 Do you think this icon (transparent background and white text) will be well displayed in cozy/cozy-collect GUI?

![](https://raw.githubusercontent.com/konnectors/digitick/5e55081a61426e5432778ce3ddb8d9e79a9e83b7/assets/icon.png)

Reply:
It will be displayed but we will not be able to see the white part.
Generally, companies are providing alternative logos for white background, do Digitick does ?

Reply:
No they don't or more precisely I did not find anything (I found this logo in the header of the main site).

Reply:
Ok they do
![](https://static.digiposte.fr/share/public/images/senders/alt/5548140.png)

Reply:
Or this another website creates it using a merge between the digitick platform logo and the digitick group logo you can find here https://commons.wikimedia.org/wiki/File:Digitick_group.png#/media/File:Digitick_group.png

Reply:
Glossy version are pretty ugly, I think this color set could be beter http://www.digiticksport.com/wp-content/uploads/2016/04/Digitick_sport.png
![digitick_sport](https://user-images.githubusercontent.com/776764/40543873-9400d336-6026-11e8-932e-2009b16112ae.png)

Reply:
We allow ourselves to change the official logo of a third party, don't we?

Reply:
![](https://i.imgur.com/rUIq15b.png)

Reply:
Donc on part sur 

![](https://is5-ssl.mzstatic.com/image/thumb/Purple128/v4/af/61/be/af61be5a-d307-b0c1-aad8-4b9f0cdb7d62/AppIcon-1x_U007emarketing-85-220-0-6.png/246x0w.jpg)